client/src/utils/parser/BpmnToSsotParsing/BpmnToSsotParsing.js

```diff
@@ -187,18 +193,20 @@ const parseBpmnToSsot = async (bpmnXml, robotId) => {
   if (typeof bpmnActivities === 'undefined') bpmnActivities = [];
 
   const bpmnStartEvent = bpmnJson['bpmn2:definitions']['bpmn2:process'][0]['bpmn2:startEvent'];
+  console.log(bpmnStartEvent)
   const bpmnEndEvent = bpmnJson['bpmn2:definitions']['bpmn2:process'][0]['bpmn2:endEvent'];
-  const bpmnShapes = bpmnJson['bpmn2:definitions']['bpmn2:process'][0]['bpmn2:startEvent']
-    .concat(bpmnJson['bpmn2:definitions']['bpmn2:process'][0]['bpmn2:task'])
+  const bpmnShapes = bpmnJson['bpmn2:definitions']['bpmn2:process'][0]['bpmn2:task']
+    .concat(bpmnJson['bpmn2:definitions']['bpmn2:process'][0]['bpmn2:startEvent'])
     .concat(bpmnJson['bpmn2:definitions']['bpmn2:process'][0]['bpmn2:endEvent'])
 
   let elementsArray = findElements(flows, bpmnShapes);
   elementsArray = enrichInstructionElements(elementsArray, bpmnActivities);
   elementsArray = enrichMarkerElements(elementsArray, bpmnStartEvent, bpmnEndEvent);
+  console.log(elementsArray)
```

Comment:
Another logging statement. Is this really needed?

Reply:
Fixed